deploy/ansible/roles-sap/5.1-dbload/tasks/main.yaml

```diff
@@ -140,20 +146,10 @@
       async:                           7200
       poll:                            60
 
-    - name:                            "DBLoad Install -Wait for asynchronous job to end"
```

Comment:
We need this to be here - Azure DevOps tasks will otherwise fail if they run for longer than 60 minutes

Reply:
But due to the poll and async in the previous task it stays there until it's either finished, or if 2 hours have passed.
In our case it's finished within those 2 hours and that results in the next task (the one I removed) failing, because the job cannot be found.

As far as I'm aware either the poll in the async task needs to be 0, or the wait needs to be removed

Reply:
Hi, I've based mine on this: https://docs.ansible.com/ansible/latest/user_guide/playbooks_async.html 

and have been using it without any issue
<img width="589" alt="image" src="https://user-images.githubusercontent.com/25486924/174020395-e3071f77-74ed-41a5-af69-ec78a86923ae.png">

Without it our Azure DevOps tasks fail - I assume that is because if there is no action on the main thread

Reply:
Hi Kimmo I just ran some tests locally with the following code: 

```yaml
- name: Async sleep for 15 seconds
  ansible.builtin.command: sleep 15
  register: results
  async: 60
  poll: 10

- name: check status (wait task)
  ansible.builtin.async_status:
    jid: "{{ results.ansible_job_id }}"
  register: job_result
  until: job_result.finished
  retries: 90
  delay: 10
  when: results.ansible_job_id is defined
 ```
 With the poll in the first task i get the following output:
 ```
 TASK [Async sleep for 15 seconds] *************************************************************************************************************************************************************************************************
ASYNC POLL on localhost: jid=317019997408.17994 started=1 finished=0
ASYNC OK on localhost: jid=317019997408.17994
changed: [localhost]

TASK [check status] ***************************************************************************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"ansible_job_id": "317019997408.17994", "attempts": 1, "changed": false, "finished": 1, "msg": "could not find job", "results_file": "/home/aa618475/.ansible_async/317019997408.17994", "started": 1, "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}

PLAY RECAP ************************************************************************************************************************************************************************************************************************
localhost                  : ok=1    changed=1    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
```

The async task is already finished when we get to the wait task, so there is no job to wait for anymore.
The other option is to set the poll to 0 to go straight to the wait task. That will result in the following output:

```
TASK [Async sleep for 15 seconds] *************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [check status] ***************************************************************************************************************************************************************************************************************
FAILED - RETRYING: [localhost]: check status (90 retries left).
FAILED - RETRYING: [localhost]: check status (89 retries left).
changed: [localhost]

PLAY RECAP ************************************************************************************************************************************************************************************************************************
localhost                  : ok=2    changed=2    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
```

Reply:
ok - let me run the same using our pipelines